python/lsst/pipe/tasks/htmIndexEngine.py

```diff
@@ -133,6 +133,135 @@ def run(self, parsedCmd):
                 result = result,
             )
 
+class IngestIndexedReferenceTask(pipeBase.CmdLineTask):
+    ConfigClass = IngestIndexedReferenceConfig
+    RunnerClass = IngestReferenceRunner
+    _DefaultName = 'IngestIndexedReferenceTask'
+
+    _flags = ['photometric', 'resolved', 'variable']
+
+    @classmethod
+    def _makeArgumentParser(cls):
+        parser = pipeBase.ArgumentParser(name=cls._DefaultName)
```

Comment:
`InputOnlyArgumentParser` doesn't insist that you provide a `--rerun` or `--output`.  Since one probably wants the reference catalog available regardless of the output, that means putting them in at the top-level, so no `--rerun` or `--output`.